bouquet/io.py

```
@@ -145,3 +147,46 @@ def add_entry(coords, atoms: Atoms, energy: float) -> None:
+    ]
+
+    with (out_dir / "ensemble_final.xyz").open("w") as fp:
+        for atoms, rel_kcal, weight in rows:
+            simple_write_xyz(
+                fp, [atoms], comment=f"E_rel={rel_kcal:.3f} kcal/mol  pop={weight:.3f}"
+            )
+
+    with (out_dir / "ensemble.csv").open("w") as fp:
+        writer = DictWriter(fp, ["index", "rel_energy_kcal", "weight"])
+        writer.writeheader()
+        for i, (atoms, rel_kcal, weight) in enumerate(rows):
```

Ruff (0.15.14)

[warning] Loop control variable `atoms` not used within loop body

(B007)